tests/mock_tables/dbconnector.py

```diff
@@ -50,7 +64,16 @@ def __init__(self, *args, **kwargs):
             raise ValueError("Invalid db")
         self.pubsub = MockPubSub()
 
-        fname = os.path.join(INPUT_DIR, fname)
+        # unix_socket_path in database_global file reflects
+        # the namespace in case of multi-namespace unit-test.
+        # This is unit-test specific string to help choose the
+        # directory to load the namespace specific json files.
+        sock_path = kwargs['unix_socket_path']
+        print(sock_path)
+        if sock_path == None or sock_path == "/var/run/redis/redis.sock":
```

Comment:
>if sock_path == None or sock_path == "/var/run/redis/redis.sock": [](start = 8, length = 65)

This seems a hack to determine whether mocking multiple namespace. Could you check `SonicDBConfig` and make decision?

Reply:
Modified as discussed. Mocked SonicV2Connector class __init__ function to add  new parameter in kwargs so that it can be used in SwssSyncClient.